### PR TITLE
Add html_icon method to piece subclasses, add piece.html_icon to view.

### PR DESCRIPTION
**Pull request by fleawig**
Pieces now drawn in place instead of listed.
Added one-line html_icon method to Piece subclasses; gets called in view.

### CODE REVIEW

**Review by nrabbi — APPROVED** [review]
Looks Fine. Approved!

File: app/services/chess_piece.rb [hunk 1]
```diff
@@ -1,10 +1,11 @@
 class ChessPiece
   attr_accessor :position
   attr_reader :color
+
   BOARD_START = 0
   BOARD_END = 7
 
-  def initialize(color, position)
+  def initialize(color, position, html_icon=nil)
```
Comment (rongg) [inline app/services/chess_piece.rb]: Is html_icon used anywhere? Looks like it's a property of the subclass which is correct, but I don't see a need for it in base class.

**Review by rongg — COMMENTED** [review]
Looks good :)

**Comment by fleawig** [comment]
@rongg That's absolutely right, no need in base class. I think I left it there from when I at first tried to do it that way. Thanks!
Since everything else has been approved I am going to merge this branch.

**Pull request merged**